addOns/ascanrulesBeta/CHANGELOG.md

```diff
@@ -6,6 +6,7 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
 ## Unreleased
 ### Change
 - Maintenance Changes.
+- Backup File Disclosure: don't raise issues for non success codes unless at LOW threshold
```

Comment:
"non-success" throughout?

Also a period here would be good.

Reply:
And add issue reference?